plugins/rapture/nexus-coreui-plugin/src/main/resources/static/rapture/NX/coreui/controller/Blobstores.js

```diff
@@ -146,12 +151,7 @@ Ext.define('NX.coreui.controller.Blobstores', {
     var me = this,
         win = form.up('nx-coreui-blobstore-add');
 
-    if (win) {
-      me.loadStoreAndSelect(action.result.data.id, true);
-    }
-    else {
-      me.loadStore(Ext.emptyFn);
-    }
+    me.getStore('NX.coreui.store.Blobstore').load();
```

Comment:
when you are un the NX.coreui namespace, then me.getStore('Repository') should work no?